### PR TITLE
we worked on navbar, log in, fixed vendor functionality to add new listing

### PR DESCRIPTION
**Pull request by kimba-1234**
1. Nav bar with login functionality was added
2. With @user, we have to differentiate between client vs vendor to be able to add new listing .  A client shouldnt be allowed to add listing, a vendor can add listing . 
3. Fixed root home to point to listings#index

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <%= render "shared/navbar" %>
```
Comment (milliepratley) [inline app/views/layouts/application.html.erb]: I could be wrong here but doesn't the navbar need to be in the body and not the head?

File: app/views/listings/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,6 @@
-<h1>Listings#index</h1>
-<p>Find me in app/views/listings/index.html.erb</p>
+<h1>Bali Sea Tours by Phil Boss</h1>
```
Comment (milliepratley) [inline app/views/listings/index.html.erb]: This line needs to be dynamic and not hard coded

**Review by milliepratley — CHANGES_REQUESTED** [review]

File: app/views/listings/index.html.erb (hunk 2 above)
Comment (kimba-1234) [reply app/views/listings/index.html.erb]: isnt this just the header 1 for the landing page ? so it will be ok to hard code the title for the home page

Comment (kimba-1234) [reply app/views/listings/index.html.erb]: It's just a random text . feel free to change h1 for the page

**Review by milliepratley — APPROVED** [review]

**Review by milliepratley — APPROVED** [review]

**Pull request merged**